### DISCUSSION TITLE
When is the newest xrdp version coming to Linux Ubuntu (23.10)?

### DISCUSSION THREAD

**Discussion by Chuck321123** [neutrinolabs/xrdp#2902] [Q&A]
So i have a raspberry pi and i run ubuntu 23.10. As xrdp randomly crashes about once in a month for me i wanted to update to the latest version to see if the problem disappears, but by doing ```sudo apt upgrade xrdp``` i get that i have the latest version available which is 0.9.21.1-1. Isnt this an old update? Any idea when the newest update comes to apt packages?

**Comment by matt335672** [comment]
That's not how Ubuntu works. xrdp only gets security patches on this platform. That's an Ubuntu decision and nothing to do with this project. They will provide later versions as part of later Ubuntu releases.

Please check existing issues to see if yours has already been reported. If it hasn't, consider reporting it yourself to help the community in general.

**Reply by Chuck321123** [reply]
@matt335672 I see. Do you have a guide to download the latest version of xrdp to ubuntu?

**Reply by matt335672** [reply]
Not a guide as such. You can try downloading and building the sources yourself by using the [wiki instructions](https://github.com/neutrinolabs/xrdp/wiki/Building-on-Debian-8) although these haven't been updated in a while.

This can be quite a time-consuming process however as you will rediscover all the issues that have been solved by the packagers already. Also, there is no guarantee your problem has been solved anyway. If we don't get fault reports we don't generally fix things.

You can see exactly what has been solved in later releases by reading the release notes [here](https://github.com/neutrinolabs/xrdp/releases)
